paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -436,7 +436,9 @@ class PaymentSheet internal constructor(
          */
         val preferredNetworks: List<CardBrand> = emptyList(),
 
-        internal val allowsRemovalOfLastSavedPaymentMethod: Boolean = true
+        internal val allowsRemovalOfLastSavedPaymentMethod: Boolean = true,
+
+        internal val paymentMethodOrder: List<String> = emptyList(),
```

Comment:
It looks like emptyList() is manually set as the default in a few places. It could be nice to define the default in just one place (e.g. in PaymentMethodOrderSettingDefinition) and then reuse that definition in all these spots. That way if we ever want to update the default, we only have to update one place

Reply:
This is a good callout. We don't do a great job of this defaulting in PaymentSheet or CustomerSheet today. We don't really have a good place to do it.

Reply:
Maybe we can do a follow up with a new internal type called `ConfigurationDefaults`, where we set these for both CustomerSheet and PaymentSheet. Thoughts?

Reply:
that sounds good to me!